generated-code-samples-for-review/language/language_sentiment_text.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Code generated by protoc-gen-go_gapic. DO NOT EDIT.
+
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"log"
+
+	language "cloud.google.com/go/language/apiv1"
+	languagepb "google.golang.org/genproto/googleapis/cloud/language/v1"
+)
+
+// [START language_sentiment_text]
+
+// sampleAnalyzeSentiment: Analyzing Sentiment in a String
+//
+// textContent: The text content to analyze
+func sampleAnalyzeSentiment(textContent string) error {
+	ctx := context.Background()
+	c, err := language.NewClient(ctx)
```

Comment:
We usually use `client` as the variable name for clients. Easy to forget what `c` is.

Reply:
Easy to change!